client/state/notices/reducer.js

```diff
@@ -47,6 +48,24 @@ export function items( state = [], action ) {
 	return state;
 }
 
+export function clicked( state = null, action ) {
```

Comment:
Just to make sure I understand, this part of the state is holding the ID of the latest clicked notice? We can maybe think of a more descriptive name, since it took me a while to figure this out.